en/generate-app/generate-app-overview.md

```diff
@@ -33,49 +33,49 @@ Right now AppBuilder generates application for Angular and Blazor. Below is a li
 |------------------------|--------------------|--------------------|
 | **Layouts**                                                      |
 | Absolute Layout        | :heavy_check_mark: | :heavy_check_mark: |
-| Card                   | :heavy_check_mark: | :question:         |
+| Card                   | :heavy_check_mark: | :warning:          |
```

Comment:
Still doesn't look quite right, a warning doesn't convey "available, but limited", I was going to suggest a yellow tick with asterisk `✔*` or something similar, but it can't be colored, so alternatively perhaps - :heavy_check_mark::construction:?

PS: Also we should totally consider the use of those emoji since they differ per platform and frankly look terrible on Android for example:
![image](https://user-images.githubusercontent.com/3198469/159042571-d902a666-a030-49e9-b607-20c1b89bb41a.png)

Reply:
[Here ](https://infragisticsinc297-my.sharepoint.com/:w:/g/personal/mvenkov_infragistics_com/EbgnJjx6_HBKhc7MAxVaNqgB3aZMjPW65cr1yI7RNrJpgQ?e=XRLR1j)is document with all the limitations I've found. Not all of these are put in the documentation.